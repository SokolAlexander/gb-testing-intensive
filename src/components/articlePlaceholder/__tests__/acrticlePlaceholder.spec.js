import React from 'react';
import { render } from '@testing-library/react';

import PlaceHolder from '../index';

describe('articles placeholder test', () => {
  it('matches snapshot', () => {
    const component = render(
      <PlaceHolder />
    );

    expect(component).toMatchSnapshot();
  });
});
