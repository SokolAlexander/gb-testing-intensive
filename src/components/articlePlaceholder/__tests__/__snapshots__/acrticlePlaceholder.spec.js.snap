// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`articles placeholder test matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="post-empty"
      >
        <div
          class="header"
        >
          HEADER OF THE TITLE
        </div>
        <div
          class="image"
        >
          IMAGE WILL GO HERE
        </div>
        <div
          class="summary"
        >
          SUMMARYSSSS
        </div>
        <div
          class="bottom"
        >
          <div
            class="source"
          >
            SOURCE
          </div>
          <div
            class="date"
          >
            DATE
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="post-empty"
    >
      <div
        class="header"
      >
        HEADER OF THE TITLE
      </div>
      <div
        class="image"
      >
        IMAGE WILL GO HERE
      </div>
      <div
        class="summary"
      >
        SUMMARYSSSS
      </div>
      <div
        class="bottom"
      >
        <div
          class="source"
        >
          SOURCE
        </div>
        <div
          class="date"
        >
          DATE
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
